### DISCUSSION TITLE
Release v1.4.1 / 5.59.1

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2303] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build improves the SandMan UI a lot and adds various new UX oriented features, it also makes the Sandboxie Plus code base Qt6 compatible, that said, the build currently uses still Qt5, a switch to Qt6 is planned for later. With regard to Qt6's no longer present Windows 7 support, this won't be an issue as a custom build of Qt6's base components will be used which are Windows 7 compatible.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

# Changelog

### Added
- added dark title bar support for Windows 11 [#2299](https://github.com/sandboxie-plus/Sandboxie/pull/2299)

### Changed
- in Sbie 5.28 and later WinInetCache is open, which breaks IE's source view, therefore it can now be disabled with 'CloseWinInetCache=y'

### Fixed
- fixed WarnProcess and WarnFolder not working with certain configurations

### Added
- added integrated run from start menu [#1836](https://github.com/sandboxie-plus/Sandboxie/issues/1836)
- added start menu enumeration [#1570](https://github.com/sandboxie-plus/Sandboxie/issues/1570)
- added UI for breakout processes [#1904](https://github.com/sandboxie-plus/Sandboxie/issues/1904)
- added option to customize double-click action per sandbox
- added new miscellaneous tab in the advanced options tab allowing to configure specific processes and other advanced options
- added "SeparateUserFolders=y" and "SandboxieLogon=y" to the sandbox options dialog
- added icons to the section labels on the option pages

### Changed
- prepared for Qt 6.3.1
- restructured the general settings page
- restructured the sandbox options page, added a new tab and moved some of the advanced options there

### Fixed
- fixed menu bar issue in Plus UI [#2280](https://github.com/sandboxie-plus/Sandboxie/pull/2280) (thanks okrc)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.4.1'>Release v1.4.1 / 5.59.1</a>.</em>

**Comment by lwcorp** [comment]
> This is a unsigned test build (that is the driver is signed but not the installers).

- [x] That's not true, it **is** signed plus it's marked as a stable release, can you fix the aforementioned statement?
![image](https://user-images.githubusercontent.com/1773306/194621649-b8266328-171a-4f1e-8483-84062718d4ed.png)

**Comment by abet00** [comment]
Every time I run Sandboxie's most recent version, the most recent version of MS Edge, it crashes and fails to load the browser/ extension. I've tried every repair imaginable, including reinstalling, deleting my settings files, and installing the previous version, but the problem persists.

Microsoft Edge
Version 106.0.1370.37 (Official build) (64-bit)

**Reply by isaak654** [reply]
See #2325

**Reply by abet00** [reply]
Thanks bro holy macaronni!! im scratching my head for the whole day trying for the fix! cheers
